packages/components/src/tour_tip/tour_tip.scss

```diff
@@ -363,7 +363,7 @@
 
     &__backdrop {
         position: absolute;
-        z-index: 999;
+        z-index: 100;
```

Comment:
this z-index was so high that was causing the issue related to hovering on any of the buttons that used the react-bootstrap tooltip (https://github.com/mattermost/mattermost-server/issues/21669#issuecomment-1327800897), by setting it just above the other page elements z-index but below the value of the rb-tooltip, the issue is fixed.